packages/botonic-core/src/router.test.js

```diff
@@ -225,6 +225,14 @@ describe('Process input (v<0.9)', () => {
       'Flow1.2'
     )
   })
+  test('old protocol: path payload input, 2nd level route with lastRoutePath', () => {
```

Comment:
please extend the PR description to explain what was the bug.
Also, I don't understand where is lastRoutePath used here